### DISCUSSION TITLE
automatic file lookup

### DISCUSSION THREAD

**Discussion by tumblebone** [mdelobelle/metadatamenu#433] [Q&A]
Hello =)

I am trying to assign a link to a "parent" note to the "parent" property of a "variant" class, based on the variant's note title being contained in the parent's aliases using the following Dataview query:

**dv.pages().where(p => p.file.aliases.includes(current.file.name));**

I'd like to avoid having to manually run an update on the "parent" property each time I create a variant class object.
How should I got about this? I have tried using the Dataview statement using a "lookup query" field type with the auto-update option but get no result.

Appreciate any help! Have a great day.

this is the parent:
![image](https://github.com/mdelobelle/metadatamenu/assets/82957949/1a6b2731-ae93-4eea-a90b-1efac054ba34)

and this is the variant:
![image](https://github.com/mdelobelle/metadatamenu/assets/82957949/e7b56488-2412-45b9-becb-3ef3997bf57e)

![image](https://github.com/mdelobelle/metadatamenu/assets/82957949/ff5a391c-2bc7-4fb4-aa1a-6bafccb96ea6)

![image](https://github.com/mdelobelle/metadatamenu/assets/82957949/cbf4e7c4-ca8f-415c-83ef-c45983a50e3a)

![image](https://github.com/mdelobelle/metadatamenu/assets/82957949/fb46df91-edbc-485a-becc-4d5e10997744)

![image](https://github.com/mdelobelle/metadatamenu/assets/82957949/5b799c3e-fa41-41df-89f8-653b0518c131)

this returns nothing:

![image](https://github.com/mdelobelle/metadatamenu/assets/82957949/c8cd87fa-cc2a-4e64-8880-2a47e27291c5)

**Comment by mdelobelle** [comment]
In order to do that, you should have a field in Parent1 containing a link to variant. 
e.g. "title: "[[variant1]]"
